supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -482,16 +483,17 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
     tellMetadata("preemptible", preemptible)
 
     // TODO: PBE: Trace callers of "new CallContext()". Seems to be multiple places in JES, etc. For now:
-    tellJesFileMetadata("stdout", jesCallPaths.stdoutFilename)
-    tellJesFileMetadata("stderr", jesCallPaths.stderrFilename)
-    tellJesFileMetadata("backendLogs:log", jesCallPaths.jesLogFilename)
+    tellMetadata("stdout", jesCallPaths.stdoutPath.toAbsolutePath)
+    tellMetadata("stderr", jesCallPaths.stderrPath.toAbsolutePath)
+    tellMetadata("backendLogs:log", jesCallPaths.jesLogPath.toAbsolutePath)
```

Comment:
While touching these lines, it should be changed to use the "transactional insert of several metadata events" implemented elsewhere in this PR.